vue.config.js

```diff
@@ -83,7 +83,7 @@ module.exports = {
         .use("file-loader")
         .loader("file-loader")
         .tap(options => Object.assign(options, {
-              name: 'img/[name]-[hash:6]-[ext]',
+              name: 'img/[name]-[hash:6]-[ext].html',
```

Comment:
This extension could be extracted into a variable to DRY it up.  This would reinforce the intent that there is a single expected fake extension to get around environmental limitations.

Reply:
In retrospect, this is a very small thing and not worth the trouble.